.drone.local.yml

```diff
@@ -5,7 +5,7 @@ clone:
 
 build:
   integration-test:
-    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.24}
+    image: $${TEST_BUILD_IMAGE=gcr.io/eminent-nation-87317/vic-integration-test:1.26}
```

Comment:
don't jump the version please, any other PRs need to rebase and update when this goes in and the PR using 25 is not going in just yet anyways.